cli/npm/managed/resolvers/local.rs

```diff
@@ -849,15 +851,61 @@ async fn sync_resolution_with_fs(
     }
   }
 
+  {
+    let packages_with_deprecation_warnings =
+      packages_with_deprecation_warnings.lock();
+    if !packages_with_deprecation_warnings.is_empty() {
+      log::warn!(
+        "{} Following packages are deprecated:",
+        colors::yellow("Warning")
+      );
+      let len = packages_with_deprecation_warnings.len();
+      for (idx, (package_id, msg)) in
+        packages_with_deprecation_warnings.iter().enumerate()
+      {
+        if idx != len - 1 {
+          log::warn!(
+            "┠─ {}",
+            colors::gray(format!("npm:{:?} ({})", package_id, msg))
+          );
+        } else {
+          log::warn!(
+            "┗─ {}",
+            colors::gray(format!("npm:{:?} ({})", package_id, msg))
+          );
+        }
+      }
+    }
+  }
+
   if !packages_with_scripts_not_run.is_empty() {
-    let packages = packages_with_scripts_not_run
+    log::warn!("{} Following packages contained npm lifecycle scripts ({}) that were not executed:", colors::yellow("Warning"), colors::gray("preinstall/install/postinstall"));
+
+    for (_, package_nv) in packages_with_scripts_not_run.iter() {
+      log::warn!("┠─ {}", colors::gray(format!("npm:{package_nv}")));
+    }
+
+    log::warn!("┃");
+    log::warn!(
+      "┠─ {}",
+      colors::italic("This may cause the packages to not work correctly.")
+    );
+    log::warn!("┠─ {}", colors::italic("To run lifecycle scripts, use the `--allow-scripts` flag with `deno install`"));
+    log::warn!("┃");
+    log::warn!("┗─ If you want to run lifecycle script, run:");
```

Comment:
I think we only really need one of these, they sort of say the same thing.

Maybe just add a colon to the first line
```
To run lifecycle scripts, use the `--allow-scripts` flag with `deno install`:
```
then print the command?

Reply:
Sure, that's reasonable